python/dcr_template.py

```diff
@@ -25,21 +25,24 @@
 This is a work in progress!
 """
 from __future__ import print_function, division, absolute_import
+from functools import reduce
 import imp
 import numpy as np
 from scipy import constants
 from scipy.ndimage.interpolation import shift as scipy_shift
 import scipy.optimize.nnls as positive_lstsq
 
 import lsst.daf.persistence as daf_persistence
-import lsst.afw.geom as afwGeom
+# import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.meas.algorithms as measAlg
 import lsst.pex.policy as pexPolicy
 # from lsst.pipe.base.struct import Struct
 from lsst.sims.photUtils import Bandpass, PhotometricParameters
 from lsst.utils import getPackageDir
 imp.load_source('calc_refractive_index', '/Users/sullivan/LSST/code/StarFast/calc_refractive_index.py')
```

Comment:
Yes, that was very ugly. I've included the calc_refractive_index code in this repo, and now just do a relative import.